utils/grpc/setup_server.go

```diff
@@ -62,7 +62,6 @@ func (s *GrpcServer) GrpcServer(port string, dbURL string, rdURL string, server
 	s.lis = lis
 
 	s.CreateServer(s.serverName, options...).
-		CreateHealthServer().
```

Comment:
maybe also remove this function entirely and `healthServer` field so there will be no confusion in the future?